### DISCUSSION TITLE
Install on VM HYPER-V network manager wait online

### DISCUSSION THREAD

**Discussion by stephane3434** [dat-linux/community#10] [Issues]
Hello, 

i want install the iso on a VM in Hyper-V

I can't define the IP adress before the system is installed

at the begining i have this message : 

[FAILED] failed to start Network Manager Wait Online

On the same server i have installed a Ubuntu  22.04 in another vm : No problem

Do you have an idea to skip this and install dat-linux ? 

Have a nice day.

**Comment by dat-linux** [comment]
Hi,

Can you be more specific. At what point in the install process do you see this? Please also attach screen shots.

(This looks like an error you see when the network interface card (virtual one in your case) is not responding. It's basically not seeing the network, which indicates a settings issue on the VM.)

**Comment by stephane3434** [comment]
Hello, 

it's the begin of the install

1) 
![image](https://user-images.githubusercontent.com/5128215/204130180-2ad61990-77b6-4fee-99bb-f203c8107f45.png)

2)
I see 
![image](https://user-images.githubusercontent.com/5128215/204130189-3ef461a3-7c31-4098-bf82-06eeab6c5d65.png)


3)

and 2 minutes after

![image](https://user-images.githubusercontent.com/5128215/204130132-15128484-0495-4dd9-a884-c7c21e46abfe.png)

that's all 

4)
on the same serveur the ubuntu vm with no specific configuration 

![image](https://user-images.githubusercontent.com/5128215/204130439-7ed0a213-c4d0-4d8f-bfa0-f817e7d6ba8a.png)

**Reply by dat-linux** [reply]
I'll need to try and reproduce it, but it looks like a situation specific to the VM and/or your machine. It's not seeing the network interface for whatever reason. 

In the meantime I suggest trying with Virtualbox and see if that works. Or install as a dual boot in a partition.

**Comment by stephane3434** [comment]
I download the iso on my windows pc and i look on Hyper-v if i obteins or not the same result

**Comment by stephane3434** [comment]
on my PC with Hyper-v

![image](https://user-images.githubusercontent.com/5128215/204157255-5b4b80f7-fc91-494c-b363-669513e52ab4.png)




boot
![image](https://user-images.githubusercontent.com/5128215/204157282-6ec48c16-2d2a-4173-ab62-09cb31654f87.png)

I have the same 

![image](https://user-images.githubusercontent.com/5128215/204157723-62e0fb7e-a9f4-4a22-aaca-a57d952ff25a.png)

I think that the iso "don't work" with Hyper-V

You can take every pc with Hyper-v and you reproduice this error

**Reply by dat-linux** [reply]
that's not showing the FAILED message from before.

Recommend to use Virtualbox and see how it goes.

**Reply by dat-linux** [reply]
When you see the splash, hit "Esc" and it will display the terminal output. Here is an example of a working startup in Virtualbox: [https://drive.google.com/file/d/1_P1IYp26MOG9ekY3T4aerInEv8Gx3WyZ/view?usp=sharing](https://drive.google.com/file/d/1_P1IYp26MOG9ekY3T4aerInEv8Gx3WyZ/view?usp=sharing)

**Comment by dat-linux** [comment]
A point release (DAT Linux 1.0.1) will be issued soon. If you'd like to get a beta of it to test out in HYPER-VM, that would be good and I can know this issue is resolved.

See https://datlinux.com/support/ for contact info.

Thanks.

**Reply by stephane3434** [reply]
perfect as soon as it is available I will test on Hyper-v
